src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -175,6 +177,20 @@ internal interface ISyntaxFactsService : ILanguageService
         bool IsLeftSideOfDot(SyntaxNode node);
         SyntaxNode GetRightSideOfDot(SyntaxNode node);
 
+        /// <summary>
+        /// Get the node on the left side of the dot if given a dotted expression. 
+        /// </summary>
+        /// <param name="allowImplicitTarget">
+        /// In VB, we have a member access expression with a null expression, this may be one of the
+        /// following forms:
+        ///     1) new With { .a = 1, .b = .a      .a refers to the anonymous type
+        ///     2) With obj : .m                   .m refers to the obj type
```

Comment:
do you test thsi for extension method import?

Reply:
I have made the change so we don't support this scenario, otherwise we'd need to check whether it's on LHS or RHS, would be fairly simple, but given how rare this is, I'd prefer to avoid the extra complexity here.

Reply:
let me rephrase :)   can we have a test to make sure we don't *crash* here :)

Reply:
Yes, we have a test for this